source/includes/app.md

```diff
@@ -115,5 +115,186 @@ app.sai.teleport();
 ```javascript
 app.sai.animate('Happy');
 ```
-
 The `sai` object can be used to control the Spatial Artificial Intelligence familiar. Use SAI to communicate essential information about a scene or direct a player's attention toward an important element.
+
+
+> Changes visibility of SAI
+
+```javascript
+app.sai.schema.setBool('visible', false);
+```
+> Changes font size of SAI text
+
+```javascript
+app.sai.schema.setNumber('fontSize', 70f);
+```
+> Changes the magnet tolerance of sai when it is unfocused. SAI will readjust its position only when the gaze spot is beyond the specified threshold.  
+
+```javascript
+app.sai.schema.setNumber('magnetTolerance', 2f);
+```
+
+> Changes text alignment of SAI
+
+```javascript
+app.sai.schema.setString('text.alignment', 'MidLeft');
+```
+
+###SAI Schema
+
+Many aspects of SAI can be adjusted through its schema. Here is a list of all of its schemas and a brief description:
+
+**Booleans**
+
+`visible`: toggles visibility
+
+`attract.enabled`: toggles whether SAI is attracted to camera
+
+`logging`: toggles SAI state change logging
+
+**Numbers**
+
+`fontSize`: sets the font size for SAI's text display
+
+`lineSpacing`: sets the line spacing for SAI's text display 
+
+`text.width`: sets the width for SAI's text display 
+
+`text.height`: sets height for SAI's text display 
+
+`text.transitionDuration`: sets the transition duration between SAI text changes
+
+`attract.delay`: sets how long SAI waits before the user's focus
+
+`attract.pause`: sets how long SAI waits before returning to the waypoint
+
+`springConstant`: sets how fast SAI moves
+
+`magnetTolerance`: sets the threshold distance that when surpassed, SAI repositions itself in front of the camera
+
+**Strings**
+
+`font`: sets the font for SAI's text display 
+
+`lineSpacing`: sets the line spacing for SAI's text display 
+
+`text.horizontalOverflow`: sets the horizontal overflow(*Overflow* or *Wrap*) for SAI's text display 
+
+`text.verticalOverflow`: sets the vertical overflow(*Overflow* or *Wrap*) for SAI's text display 
+
+`text.alignment`: sets the alignment for SAI's text display 
+
+**Colors**
+
+`text.shadow.color`: sets the shadow color for SAI's text display 
+
+**Vectors**
+
+`text.shadow.offset`: sets the shadow offset for SAI's text display 
+
+
+> Adds button option to sai for users to choose from
+
+
+##SAI Buttons(Preview)
+
+The `prompt` command can be used to let SAI prompt button options for the player to choose from. The first argument is a callback, the second is the question, the following are chained questions.
+
+`prompt(Callback callback, String Question, String option1, String option2, ...)`
```

Comment:
Hard because js doesn't really have a ton of type support natively - but I'd change `Callback` -> `Function` to match better. Still perfectly fine to have the arg name be callback though!

Reply:
Got it! Will make the changes. Thank you :)